src/contest.c

```diff
@@ -1492,7 +1492,7 @@ static void Task_DisplayAppealNumberText(u8 taskId)
 static void Task_TryShowMoveSelectScreen(u8 taskId)
 {
     // Wait for button press to show move select screen
-    if ((JOY_NEW(A_BUTTON)) || (gMain.newKeys == B_BUTTON))
+    if ((gMain.newKeys & A_BUTTON) || (gMain.newKeys == B_BUTTON))
```

Comment:
TODO: Reintroduce the JOY_XXX macro changes

Reply:
This should be done in this PR

Reply:
Done